docs/KubeFATE_command_line_user_guide.md

```diff
@@ -0,0 +1,279 @@
+# KubeFATE Command Line User Guide
+
+### What is KubeFATE command line
+
+KubeFATE command line is a command line tool that connects KubeFATE  service to deploy FATE on Kubernetes.
+
+### Install KubeFATE command line
+
+Before using kubefate command line, you need to deploy KubeFATE  service.
+
+#### Deploy KubeFATE service on Kubernetes
+
+Get the code
+
+```bash
+git clone https://github.com/FederatedAI/KubeFATE.git
+cd KubeFATE/k8s-deploy
+```
+
+Deploy KubeFATE service on Kubernetes
+
+```
+kubectl apply -f ./rbac-config.yaml
+kubectl apply -f ./kubefate.yaml
+```
+
+*A more detailed deployment process is here([deploy KubeFATE in Kubernetes](https://github.com/FederatedAI/KubeFATE/tree/master/k8s-deploy#initial-a-new-fate-deployment)).*
```

Comment:
Should add a note about `rbac-config.yaml` just a reference access setting.

Reply:
[deploy KubeFATE in Kubernetes](https://github.com/FederatedAI/KubeFATE/tree/master/k8s-deploy#initial-a-new-fate-deployment) This link contains deployment details, so I won’t focus on it here.